app/src/main/java/com/kodeco/android/coordplot/MapGraph.kt

```diff
@@ -1,30 +1,37 @@
 package com.kodeco.android.coordplot
 
+import androidx.compose.foundation.Image
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.offset
 import androidx.compose.foundation.layout.size
-import androidx.compose.foundation.shape.CircleShape
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.draw.clip
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
 
 @Composable
 fun MapGraph(xPercent: Float, yPercent: Float, mapSize: Int, cursorSize: Int) {
+    val cursorResource = painterResource(id = R.drawable.arrow)
     Box(modifier = Modifier
         .size((mapSize + cursorSize).dp)
         .background(Color.LightGray)) {
```

Comment:
```suggestion
        .background(Color.LightGray)) {
```
Also try exploring the color from the Material theme like MaterialTheme.colorScheme.primary or MaterialTheme.colorScheme.secondary